src/Fantomas/schema.json

```diff
@@ -11,7 +11,31 @@
     "SemicolonAtEndOfLine": {
       "type": "boolean"
     },
-    "SpaceBeforeArgument": {
+    "SpaceBeforeUnitArgumentInUppercaseInvocation": {
+      "type": "boolean"
+    },
+    "SpaceBeforeUnitArgumentInLowercaseInvocation": {
+      "type": "boolean"
+    },
+    "SpaceBeforeParenthesesInUppercaseInvocation": {
+      "type": "boolean"
+    },
+    "SpaceBeforeParenthesesInLowercaseInvocation": {
+      "type": "boolean"
+    },
+    "SpaceBeforeUnitParameterInUppercaseFunctionDefinition": {
+      "type": "boolean"
+    },
+    "SpaceBeforeUnitParameterInLowercaseFunctionDefinition": {
+      "type": "boolean"
+    },
+    "SpaceBeforeParenthesesInUppercaseFunctionDefinition": {
+      "type": "boolean"
+    },
+    "SpaceBeforeParenthesesInLowercaseFunctionDefinition": {
+      "type": "boolean"
+    },
```

Comment:
need update

Reply:
Updated, although maybe this also needs its own issue as I've created (https://github.com/fsprojects/fantomas/issues/725) for the documentation.